dynamiqs/result.py

```diff
@@ -188,3 +188,49 @@ class MEResult(Result):
         [Batching simulations](../../documentation/basics/batching-simulations.md)
         tutorial for more details.
     """
+
+
+class PropagatorResult(Result):
+    r"""Result of the Schrödinger equation integration to obtain the propagator.
+
+    Attributes:
+        propagator _(array of shape (..., ntsave, n, n))_: Saved propagators.
```

Comment:
```suggestion
        propagators _(array of shape (..., ntsave, n, n))_: Saved propagators.
```
(for consistency with `result.expects` and `result.states`)